FirstProject/new-evaluation.html

```diff
@@ -41,23 +39,23 @@
             <h2 class="description_section" style="color: #03A9F4;">Technical level</h2>
         </div>
 
-        <div class="row" style="background-color: #03A9F4; margin: auto; width: 73.1%; display: flex;">
+        <div class="row" style="background-color: #03A9F4; margin: auto; width: 73.2%; display: flex;">
```

Comment:
Interesting percentage for the width. Might I ask how you arrived at this particular value? Also, do the proportions still hold when resizing the browser window?